documentation/getting-started.md

```diff
@@ -35,3 +39,147 @@ this module wherever you'd like to use the components.
+NoopAnimationsModule.
+
+```typescript
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+
+@NgModule({
+  ...
+  imports: [NoopAnimationsModule],
+  ...
+})
+export class AppModule { }
+```
+
+**Note:** @angular/animations uses the WebAnimation API that isn't supported by
+all browsers yet. If you want to support animations in these browsers, you'll
+have to
+[include a polyfill](https://github.com/web-animations/web-animations-js).
+
+### Step 3: Import the component modules
+
+Import the NgModule for each component you want to use:
+
+```typescript
+import { DtButtonModule } from '@dynatrace/barista-components/button';
+import { DtSelectModule } from '@dynatrace/barista-components/select';
+
+@NgModule({
+  ...
+  imports: [DtButtonModule, DtSelectModule],
+  ...
+})
+export class PizzaPartyAppModule { }
+```
+
+Alternatively, you can create a separate NgModule that imports all of the
+Barista components that you will use in your application. You can then include
+this module wherever you'd like to use the components.
+
+**Note:** Whichever approach you use, be sure to import the barista-components
+modules after Angular's BrowserModule, as the import order matters for
+NgModules.
+
+### Step 4: Include the styles
+
+The Barista components library ships with two different variants for including
+styles. You can either import the core styles that are **required** to use the
+barista-components. These core styles are just the bare minimum for the
+component and do not change the global styling of your app.
+
+Add the following line to your project's architect options in the angular.json
+file to include core styles:
+
+```json
+  "styles": ["node_modules/@dynatrace/barista-components/style/main.scss"],
+```
+
+You can also import the second variant, which also includes general styling for
+headlines, text-formatting, etc. by including the following line:
+
+```json
+  "styles": ["node_modules/@dynatrace/barista-components/style/index.scss"],
+```
+
+### Step 5: Optional - add fonts and icon files to your assets
+
+Add this section to the `assets` array in your projects architect in the
+angular.json. This will copy all svgs from the iconpack and all fonts shipped
+with the barista-components library to your dist package and will make Angular
+aware of the file dependencies.
+
+```json
+...
+{
+  "glob": "metadata.json",
+  "input": "node_modules/@dynatrace/barista-icons",
+  "output": "/assets/icons",
+},
+{
+  "glob": "*.svg",
+  "input": "node_modules/@dynatrace/barista-icons",
+  "output": "/assets/icons"
+},
+{
+  "glob": "**/*",
+  "input": "node_modules/@dynatrace/barista-fonts/fonts",
+  "output": "/fonts"
+},
+...
+```
+
+To provide the configuration to the application, you will need to add this to
+the providers of your app module.
+
+```typescript
+providers: [
+  {
+    DtIconMoudle.forRoot({ svgIconLocation: `/assets/icons/{{name}}.svg` }),
```

Comment:
sorry I was not that clear, you don't need to specify the providers in the module decorator.

You have to write:
```typescript
@NgModule({
  declarations: [...],
  imports: [
    ...
    DtIconMoudle.forRoot({ svgIconLocation: `/assets/icons/{{name}}.svg` }),
  ],
  bootstrap: [...],
})
export class AppModule {}
```

```